server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -236,6 +335,44 @@ public ClusterMetadataManifest(
         List<UploadedIndexMetadata> indices,
         String previousClusterUUID,
         boolean clusterUUIDCommitted
+    ) {
+        this(
+            clusterTerm,
+            version,
+            clusterUUID,
+            stateUUID,
+            opensearchVersion,
+            nodeId,
+            committed,
+            codecVersion,
+            globalMetadataFileName,
+            indices,
+            previousClusterUUID,
+            clusterUUIDCommitted,
+            null,
+            null,
+            null,
+            null
+        );
+    }
+
+    public ClusterMetadataManifest(
```

Comment:
If we have a Builder for ClusterMetadataManifest, why do we need a new constructor each time we add more fields?

Reply:
Currently, we are using these constructors only while creating the Manifest object. Also, this constructor is getting called from Builder `build()` as well. Do you think we should rather move away from directly calling the constructor to using Builder?

Reply:
Yes I think it makes sense to use the builder rather than creating new constructors everytime a new field is added.

Reply:
Updated, we only have one constructor now.